ibis/pandas/execution/generic.py

```diff
@@ -372,6 +399,7 @@ def execute_aggregation_dataframe(op, data, scope=None, **kwargs):
         for metric in op.metrics
     ]
 
+    # group by always needs a reset to get the grouping key back as a column
```

Comment:
for an aggregation (not .transform), you can pass ``as_index=False`` to do this; though what you are doing here is ok (as you did document), but former is *sligthly* more idiomatic

Reply:
I remember trying to use `as_index=False` and it being hard to handle consistently downstream, in places like this. I will look into it again if I have a chance.